src/i18n/fr.json

```diff
@@ -98,6 +98,51 @@
     "notifications": "Notifications",
     "time_ago": "Il y a {{time}}"
   },
+  "offerTypes": {
+    "Concert": "Concert",
+    "Theatervoorstelling": "Théâtre",
+    "Lezing of congres": "Conférence ou congrès",
+    "Dansvoorstelling": "Spectacle de danse",
+    "Festival": "Festival",
+    "Toeristisch evenement": "Événement touristique",
+    "Film": "Cinéma",
+    "Sportwedstrijd bekijken": "Assister à une compétition sportive",
+    "Cursus met open sessies": "Cours avec sessions ouvertes",
+    "Lessenreeks": "Série de cours",
+    "Sport en beweging": "Activité sportive",
+    "Kamp of vakantie": "Camp de vacances",
+    "Begeleide rondleiding": "Visite guidé",
+    "Route": "Itinéraire",
+    "Spel of quiz": "Jeu ou quiz",
+    "Party of fuif": "Soirée",
+    "Festiviteit": "Festivité",
+    "Congres of studiedag": "Congrès ou journée d'étude",
+    "Eet- of drankfestijn": "Festin de nourriture ou de boisson",
+    "Thema of pretpark": "Parc à thème ou parc d'attractions",
+    "Theater": "Théâtre",
+    "Jeugdhuis of jeugdcentrum": "Maison de jeunes ou centre de jeunesse",
+    "Cultuur- of ontmoetingscentrum": "Centre culturel ou centre de réunion",
+    "Discotheek": "Discothèque",
+    "Bibliotheek of documentatiecentrum": "Bibliothèque ou centre de documentation",
+    "Bioscoop": "Cinéma",
+    "Horeca": "Horeca",
+    "Openbare ruimte": "Espace public",
+    "Tentoonstelling": "Exposition",
+    "Markt, braderie of kermis": "Marché, foire ou fête foraine",
+    "Natuurgebied of park": "Zone naturelle ou parc",
+    "Natuur, park of tuin": "Zone naturelle ou parc",
+    "Beurs": "Foire / exposition",
+    "Monument": "Monument",
+    "Opendeurdag": "Journée portes ouvertes",
+    "Recreatiedomein of centrum": "Centre de loisirs ou centre de récréation",
+    "Park of tuin": "Jardin ou parc",
+    "Archeologische Site": "Site Archéologique",
+    "School of onderwijscentrum": "École ou centre éducatif",
+    "Sportcentrum": "Centre sportif",
+    "Winkel": "Magasin",
+    "Museum of galerij": "Musée ou galerie",
+    "Zaal of expohal": "Hall ou salle d'expo"
+  },
```

Comment:
Nit: Maybe we should use the ids of the terms as keys here instead of their labels? Because the copy of the labels changes sometimes, and then we need to make sure we also change the copy here in the _key_ as well. While the ids stay the same.

Reply:
I'm going to implement it this way now 😄